README.md

```diff
@@ -1163,10 +1163,12 @@ Table of Contents
   * [AllTheFreeStock](https://allthefreestock.com) - a curated list of free stock images, audio and videos.
   * [Ant Design Landing Page](https://landing.ant.design/) - Ant Design Landing Page provides a template built by Ant Motion's motion components. It has a rich homepage template, downloads the template code package, and can be used quickly. You can also use the editor to quickly build your own dedicated page.
   * [Backlight](https://backlight.dev/) — With collaboration between developers and designers at heart, Backlight is a very complete coding platform where teams build, document, publish, scale and maintain Design Systems.Free plan allows up to 3 editors working on 1 design system with unlimited viewers.
+  * [Blob Generator](https://www.magicpattern.design/tools/blob-generator) — Create random organic-shaped blobs with solid colors, linear/radial gradients, and pattern fills.
```

Comment:
What is the point of this service? We've had a similar one recently and no-one could figure out why it exists and so we rejected it.  Along the same lines I am inclined to not accept this one.

Reply:
You can place them as section backgrounds, or mask images with them (instead of the traditional oval shape). But I understand if you don't want to feature them :)